src/plugins/gc-subway/gc-subway.js

```diff
@@ -39,6 +40,12 @@ var $document = wb.doc,
 
 		if ( elm ) {
 			$elm = $( elm );
+			$h1 = $( "h1", $elm );
+
+			// Add subway H1 to skip links
+			if ( !skipLink ) {
+				skipLink = wb.addSkipLink( $h1.text, { name: "href", value: "#" + $h1.id } );
```

Comment:
You will need to adjust this code based on the recent request change on the addSkipLink feature